packages/seal/src/Search/SearchBuilder.php

```diff
@@ -82,6 +91,23 @@ public function offset(int $offset): static
         return $this;
     }
 
+    /**
+     * @param array<string> $fields
+     */
+    public function highlight(array $fields, string $preTag = '<mark>', string $postTag = '</mark>'): static
+    {
+        $this->highlightFields = $fields;
+        $this->highlightPreTag = $preTag;
+        $this->highlightPostTag = $postTag;
+
+        return $this;
+    }
+
+    public function getSearcher(): SearcherInterface
+    {
+        return $this->searcher;
+    }
```

Comment:
TODO need to check why I exposed this in https://github.com/PHP-CMSIG/search/pull/483